src/vmap.js

```diff
@@ -4,7 +4,7 @@ import { childrenByName, parseXMLNode } from './parser_utils';
 class VMAP {
   constructor(xml) {
     if (
-      (xml != null ? xml.documentElement : undefined) == null ||
+      (xml !== null ? xml.documentElement : undefined) === null ||
```

Comment:
I agree that this check was weak and error prone (legacy stuff from the coffeescript to es6 conversion), and that we should improve it. 

But this change doesn't do what we want, because if `xml` is `undefined` we try to access anyway `xml.documentElement` (since `undefined !== null = true`)

we should better change it to:
```javascript
if (
  !xml ||
  !xml.documentElement ||
  !xml.documentElement.localName !== 'VMAP'
) {
...
}
```